### PR TITLE
fix(country): restore US headlines in country briefs

### PR DESCRIPTION
**Pull request by koala73**
## Summary

Country briefs now show relevant US headlines written with the common `US` and `U.S.` forms instead of falling through to the empty state. Plain `US` stays case-sensitive so the pronoun “us” is not misclassified, while the existing first-country-mentioned rule remains consistent across Top News and critical-news counts.

## Validation

The focused regression failed on both US headline forms before the fix and now passes all 3 cases, including lowercase country-code input and the negative pronoun case. The Node 24 pre-push gate also passed 90 DOM files / 747 tests, 273 Edge checks, TypeScript, architecture boundaries, safe HTML, premium-fetch parity, Unicode safety, and version sync.

---

[Compound Engineering](https://github.com/EveryInc/compound-engineering-plugin)
Codex

### CODE REVIEW

File: src/app/country-intel.ts [hunk 1]
```diff
@@ -1867,9 +1853,15 @@ export class CountryIntelManager implements AppModule {
...
-    const match = new RegExp(`(^|[^a-z0-9])${CountryIntelManager.escapeRegExp(normalizedTerm)}(?=$|[^a-z0-9])`, 'i').exec(text);
+    const trimmedTerm = term.trim();
+    if (!trimmedTerm) return -1;
+    // Plain two/three-letter country acronyms are case-sensitive so US does
+    // not match the pronoun "us". Dotted forms such as U.S. are unambiguous
+    // and remain case-insensitive with every full-name and demonym alias.
+    const caseSensitive = /^[A-Z]{2,3}$/.test(trimmedTerm);
+    const matchText = caseSensitive ? text : text.toLowerCase();
+    const matchTerm = caseSensitive ? trimmedTerm : trimmedTerm.toLowerCase();
+    const match = new RegExp(`(^|[^a-z0-9])${CountryIntelManager.escapeRegExp(matchTerm)}(?=$|[^a-z0-9])`).exec(matchText);
```
[the remark this answers is not in the document]
Comment (koala73) [reply src/app/country-intel.ts]: > Require real boundaries around case-sensitive acronyms

Addressed in 89f5533cf1: the acronym matcher now uses case-complete ASCII word boundaries. The regression proves that `USD` does not match the US and does not suppress a later China match.
